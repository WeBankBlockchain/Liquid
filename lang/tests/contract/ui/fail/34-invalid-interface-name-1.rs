use liquid_lang as liquid;

#[liquid::interface(name = "r#fn")]
mod foo {
    extern "liquid" {
        fn bar();
    }
}

fn main() {}