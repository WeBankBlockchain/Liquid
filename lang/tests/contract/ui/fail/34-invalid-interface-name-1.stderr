error: invalid interface name: `r#fn`
 --> $DIR/34-invalid-interface-name-1.rs:3:28
  |
3 | #[liquid::interface(name = "r#fn")]
  |                            ^^^^^^
